# DP: JDK-8067330, ZERO_ARCHDEF incorrectly defined for PPC/PPC64 architectures

--- a/common/autoconf/platform.m4
+++ b/common/autoconf/platform.m4
@@ -436,7 +436,8 @@
     m68k)  ZERO_ARCHDEF=M68K ;;
     mips|mipsn32|mips64)  ZERO_ARCHDEF=MIPS ;;
     mipsel|mipsn32el|mips64el)  ZERO_ARCHDEF=MIPSEL ;;
-    ppc*)    ZERO_ARCHDEF=PPC   ;;
+    ppc)     ZERO_ARCHDEF=PPC32 ;;
+    ppc64)   ZERO_ARCHDEF=PPC64 ;;
     s390*)   ZERO_ARCHDEF=S390  ;;
     sh*)   ZERO_ARCHDEF=sh  ;;
     sparc*)  ZERO_ARCHDEF=SPARC ;;
